code/modules/projectiles/guns/projectile/automatic/molly.dm

```diff
@@ -52,8 +52,8 @@
 		iconstring += "_s"
 		itemstring += "_s"
 
-	if (wielded)
-		itemstring += "_doble"
+//	if (wielded)
```

Comment:
Delete these if they're not used.

Reply:
It should be used, there is just no sprite yet.